location.md

```diff
@@ -4,17 +4,61 @@ title: "All roads lead to Rome"
 permalink: /location/
 ---
 
-<div class="backdrop">
+<div class="backdrop" style="background: url(/assets/rome/the-city.jpg) 50% 40%; background-repeat:no-repeat; padding: 0; margin-bottom: 1em;" >
   <div class="popout">
     <section>
       <h1>Welcome to {{site.location.city}}</h1>
-      <p>We are very pleased to host the fourth edition of RustFest in Rome, Italy.</p>
+      <p> We are very pleased to host the fourth edition of RustFest in Rome, Italy. </p>
     </section>
   </div>
 </div>
 
-<section>
-<p>
-Rome is the capital of Italy.
-</p>
+<section markdown="1">
 
+Rome, the eternal city, is the capital of Italy with 3 million of residents. It's located in the central-western portion of the Italian Peninsula, within Lazio, along the shores of the Tiber. 
+Rome's history spans 28 centuries, leaving behind a huge number of monuments spanning from the famous Colosseum, to Piazza Di Spagna and Fontana Di Trevi.
+Also the Vatican City is an independent country inside the city boundaries of Rome, the only existing example of a country within a city: for this reason Rome has been often defined as capital of two states.  
+
+
+</section>
+
+<section class="img-grid">
+    <img src="/assets/rome/colosseum.jpg" alt="The Colosseum Photo">
+</section>
+<br/>
+<section markdown="1">
+    
+## Travelling to Rome    
+  
+### By Plane
+Rome has two Airports the [Leonardo Da Vinci](https://www.adr.it/fiumicino) airport that sit in the nearby town of Fiumicino is the biggest of Rome and allow you to travel from most locations in Europe with flights both low cost and not, it has a direct train to the main station of Rome called [Roma Termini](https://www.romatermini.com/en/) that is walking distance from the conference venue.
+The second airport is [Rome Ciampino](https://www.adr.it/ciampino) is mostly served by low cost companies and it is connected by a bus and underground to the station Roma Termini.
+
+### By Train
+Rome has fast trains stopping at Roma Termini from most of the main cities of Italy (Naples, Turin, Milan, Venice, Bologna), but not direct international trains, so for the international travel is suggested to use flights.
+
+## Entry requirements
+Italy is part of the European Union (EU) as well as the Schengen Agreement. EU and EEA citizens can enter Italy without a visa, as long as a valid ID card or passport can be presented. As usual, a visa granted for any Schengen member is valid in Italy as well. For more information, check out Wikivoyage on Travelling around the Schengen Area.
+	 
+## Venue
+Rust Fest Rome will be hosted both days at the (Frentani Congress Center)[https://www.congressifrentani.it/centrocongressi/] and for the second day some workshops will be hosted at: [Hotel Universo](https://www.hoteluniverso.com/it/meeting-congressi/dettagli-sale-meeting-roma.aspx)
+
+## Getting there
+The address of the venue is "Via dei Frentani, 4 00185 Roma"
```

Comment:
Remove the quotes.
Either link to Google Maps (or OpenStreetMap) or make the address bold.
End the sentence with a dot.